src/test/java/com/scottlogic/filter/FilterDateTest.java

```diff
@@ -7,12 +7,13 @@
 import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
 
+import java.util.Arrays;
 import java.util.List;
 
 public class FilterDateTest {
 
-    FilterDateEquals dateEqualsFilterTest = new FilterDateEquals();
-    FilterBetweenDates dateBetweenFilterTest = new FilterBetweenDates();
+    FilterDateEquals filterDateEquals = new FilterDateEquals();
+    FilterBetweenDates filterBetweenDates = new FilterBetweenDates();
 
     @Nested
     @DisplayName("Date equals filter")
```

Comment:
Do you have a test case for before start and after end to check that it doesn't pick anything it shouldn't?

Reply:
The functionality test case already filters out 3 of the 6 posts that are outside of the range, so it does both - should I make this more obvious?

Reply:
That's acceptable, maybe update the name to make it known that it is captured.
Personally I'd make another test case when it is a different behaviour.

Given "some dates" When "filter between" Then "some verification"
When one of those parts get too big, I'll split the test case.